doc/releases/changelog-dev.md

```diff
@@ -82,20 +156,19 @@
 
   @qml.qnode(dev)
   def circuit():
-
-      generator(wires=range(3))
-      qml.AmplitudeAmplification(U, O, iters=5, fixed_point=True, work_wire=3)
+      generator(wires=range(3)) # prepares |Psi> = U|0>
+      qml.AmplitudeAmplification(U, O, iters=10)
 
       return qml.probs(wires=range(3))
-
   ```
   
   ```pycon
   >>> print(np.round(circuit(), 3))
-  [0.013, 0.013, 0.91, 0.013, 0.013, 0.013, 0.013, 0.013]
-
+  [0.01  0.01  0.931 0.01  0.01  0.01  0.01  0.01 ]
   ```
 
+  As expected, we amplify the $\vert 2 \rangle$ state.
+
```

Comment:
Note that `qml.Qubitization` has also been added, coming in just before feature freeze :rocket: